R/clean_redcap.R

```diff
@@ -109,7 +110,7 @@ clean_redcap <- function(
 #' \code{REDCapR::redcap_metadata_read()$data}
 #'
 #' @importFrom dplyr filter pull select relocate rename
-#' @importFrom tidyselect all_of everything starts_with
+#' @importFrom tidyselect all_of everything starts_with matches
```

Comment:
If you adopt my suggestion below:
```suggestion
#' @importFrom tidyselect all_of everything starts_with any_of
```